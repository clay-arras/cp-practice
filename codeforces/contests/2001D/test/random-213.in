1
17
7 13 4 7 5 15 9 16 23 12 14 14 9 13 19 21 3
