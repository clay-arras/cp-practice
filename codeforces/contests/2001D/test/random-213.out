13
13 4 7 5 15 9 16 23 12 14 19 21 3
